### ISSUE TITLE
thoughts about a theme logo?

### ISSUE THREAD

**Issue by 2bndy5** [jbms/sphinx-immaterial#264]
We don't have a branded image to use as a theme logo. I know we don't really need one, but I thought I'd ask. If this ever gets resolved (as completed), then it would be good to have a favicon that uses the same image.

When initially revising the docs from sphinx-material relics, I chose to use an animated free GIF I found online as a logo; the favicon is really just a colorized version of the material/bookshelf icon.

In an effort to jump-start this discussion, I mocked up something that has the initials "SI" that tries to follow the [MaterialUI logo](https://worldvectorlogo.com/logo/material-ui) design.
![image](https://github.com/jbms/sphinx-immaterial/assets/14963867/305eeb55-2b4b-42d9-8d9e-91ef6edca933) This uses 30% transparent fill color within the letters.
I feel like we can do better though -- maybe use shapes that are a bit more 3D like the [mkdocs-material logo](https://github.com/squidfunk/mkdocs-material/blob/1801f2f2b9100d449e0fd0ea7de69d7078e0b651/src/.icons/logo.svg).

<details><summary>3D <em>rough</em> draft</summary>

![image](https://github.com/jbms/sphinx-immaterial/assets/14963867/62226b04-47c7-48f2-8fa7-ac78e21060b9)

</details> 

My expertise with GIMP is much better than using Inkscape. The above SVG 2D image (made with Inkscape) is actually using paths extracted from an imported GIMP image. I'm not sure how the pros do it, but I imagine Inkscape isn't the only tool in the shed. And, no I haven't purchased Photoshop or anything proprietary for hobby work like this. The 3D rough draft was also made with Inkscape but needs polish once the dimensions are finalized.

The license for the logo should allow hosting the image's source file(s) (GIMP's .xcf file and/or Inkscape's  un-optimized .svg file) in this repo. I'm not familiar with any limitations inherent from using proprietary software to create images.

**Comment by tunchunairarko** [comment]
I think the logo colour can be improved, but the logo's vector is nicely designed

**Comment by 2bndy5** [comment]
> I think going with SVG is the better choice

Yeah. I'm still learning to use Inkscape though. Turns out the SVG that Inkscape uses as a project file needs to be optimized when ready to be exported for deployment. This is why I think we should have 1 SVG for edits in Inkscape and another optimized SVG for use in Sphinx (or elsewhere). The optimization could probably happen in the `npm run build` task when the theme's pkg is built for pypi.
